lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +113,24 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if not (isinstance(unknown_profile, dict)
+            and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict)):
+        return None
+
+    distance1 = compare_profiles(unknown_profile, profile_1)
+    distance2 = compare_profiles(unknown_profile, profile_2)
+
+    if (
+            isinstance(distance1, float)
+            and isinstance(distance2, float)
+    ):
+
+        if distance1 > distance2:
+            return str(profile_2['name'])
+        if distance1 < distance2:
+            return str(profile_1['name'])
+    return [profile_1['name'], profile_2['name']].sort()
```

Comment:
Are you sure? Typing says that you can return only one string

Reply:
fixed